.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Purpose
+_Describe the problem or feature in addition to a link to the issues._
+
+## Approach
+_How does this change address the problem?_
+
+## Tests for New Behavior
+_What new tests were added to cover new features or behaviors?_
+
+## Checklist
```

Comment:
Can we make this checklist imperative?
+ [ ] Signed and returned a copy of the CLA
+ [ ] Added tests for any new features or behaviors
+ [ ] Ran ``tox -e flake8`` to make sure code style is consistent
+ [ ] Built and reviewed the docs
+ [ ] Added a note to the changelog